daphne/src/auth.rs

```diff
@@ -51,11 +51,24 @@ impl AsRef<BearerToken> for BearerToken {
     }
 }
 
+#[cfg(feature = "send-traits")]
```

Comment:
I think the `conditional_sync` module is technical enough that it needs a docucomment describing its purpose.